parts/k8s/artifacts/kubernetesazurekms.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=azurekms
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=simple
+Restart=always
+EnvironmentFile=-/etc/default/%p
+TimeoutStartSec=0
+ExecStart=/usr/bin/docker run \
+  --rm \
```

Comment:
remove this. a long with `name` -- Let docker automatically generate name. This will allow us to get logs for previous start/fail events for debugging miss-behaving clusters